packages/appium/docs/en/guides/branch-testing.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Local Validation Of Driver or Plugin PRs
```

Comment:
I'd use a shorter title, maybe 'Local Testing of Extension PRs'

Reply:
changed